index.css

```diff
@@ -0,0 +1,22 @@
+*{
+    padding: 0;
+    margin: 0;
+}
+body{
+    background-color: white;
+    padding-top: 30px;
+    padding-bottom: 30px;
+    padding-right: 10px;
+    padding-left: 10px;
```

Comment:
- [ ] Please ensure to use the short format to save up more space. `padding: 0 10px;` would've been more appropriate.